src/main/java/use_case/userList/LoginUseCase.java

```diff
@@ -1,22 +1,23 @@
 package use_case.userList;
 
+import constant.fileSystem.FileLocation;
 import constant.uiMessage.LoginResult;
 import entity.User;
 import use_case.boundary.LoginInputBoundary;
+import use_case.boundary.output.LoginOutputBoundary;
 
 public class LoginUseCase implements LoginInputBoundary {
 
     /**
      * A list of users organized by id.
      */
     private final UserList users;
+    private final LoginOutputBoundary outputBoundary;
 
 
-    public LoginUseCase() {
-        users = new UserList();
-    }
-    public LoginUseCase(String filepath){
-        users = new UserList(filepath);
+    public LoginUseCase(LoginOutputBoundary loginOutputBoundary){
+        this.users = new UserList(FileLocation.USER_FILE_LOCATION);
+        this.outputBoundary = loginOutputBoundary;
```

Comment:
Should directly instantiate the Controller that implements the output boundary here, instead of passing it in as a parameter.